Altis_Life.Altis/core/functions/fn_isNumber.sqf

```diff
@@ -0,0 +1,27 @@
+/*
+    File: fn_isNumber.sqf
+    Author: Bryan "Tonic" Boardwine
+    Edited by: Daniel Stuart (Moved to client-side, got rid of KRON)
+
+    Description:
+    Return if the given string only contains numbers.
+
+    Parameter(s):
+        0: STRING - String to be analyzed
+
+    Returns:
+    BOOL
+*/
+params [
+    ["_array","",[""]]
+];
+_array = _array splitString "";
```

Comment:
We could use toArray and replace the strings below w/ just numbers instead, seems more logical. Can also use a count